index.js

```diff
@@ -12,7 +12,7 @@ function getDisplayStyle(elementType) {
     }
 }
 
-function fade(type, ms, el) {
+function fade(type, ms, el, cb) {
```

Comment:
can you add a default catch-all to cb not being set? Something like `cb = (typeof cb !== 'function' ? function(){} : cb);`